provision/testdata/flavor_cko_calico-3.23.2.inp.yaml

```diff
@@ -24,8 +24,42 @@ aci_config:
     - mykube_extepg
 
 connectivity_checker:
-
-
+ error_pods_reporting_enable: true
+ reachability_test_enable:
+  periodic_sync:
+   enable_updates: true
+   interval: 120
+  external_url:
+   url: 'google.com'
+  proxy: ""
```

Comment:
I believe we will be capturing proxy in a different config section like this: https://github.com/noironetworks/acc-provision/blob/6b4a734b8c22fc4c8869cffbb81ee8846770333e/provision/testdata/flavor_cko_aci.inp.yaml#L58-L61

So we wouldbnt need it here again.
@snaiksat, do you think we can use the same config here?